backend/src/models/item/postgres.js

```diff
@@ -48,6 +48,20 @@ const ItemRepo = (postgres) => {
       FOREIGN KEY (item_id) REFERENCES items(item_id)
     );`;
 
+  /**
+   * Set up the shipment table if it does not exist
+   * 
+   * @type {string}
+   */
+  const createShipmentTableSQL = `
+    CREATE TABLE IF NOT EXISTS shipments(
+      shipment_id SERIAL PRIMARY KEY,
+      item_id integer NOT NULL,
```

Comment:
Add foreign key here to specify that it matches up with item table. Also, should Item have a reference to shipment?